docker/horcrux/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM golang:alpine AS build-env
+FROM --platform=$BUILDPLATFORM golang:1.19.1-alpine3.16 AS build-env
```

Comment:
I usually just specify `golang:1.19-alpine` so that we automatically use the latest patch release without having to update this file.

Generally, we would want to adopt go 1.19.<next> as soon as it's available.